src/metpy/io/station_data.py

```diff
@@ -120,8 +121,17 @@ def _read_airports_file(input_file=None):
                          }).to_dict()
 
 
-class StationLookup:
-    """Look up station information from multiple sources."""
+@exporter.export
+class StationLookup(Mapping):
+    """Look up station information from multiple sources.
+
+    This class follows the `Mapping` protocol with station ID as the key. This makes it
+    possible to e.g. iterate over all locations and get all of a certain criteria:
+    >>> import metpy.io
```

Comment:
```suggestion
    possible to e.g. iterate over all locations and get all of a certain criteria:
    
    >>> import metpy.io
```
Else the code sample doesn't render and format appropriately.

Reply:
Good catch.